src/main/java/nextstep/subway/path/domain/DefaultFarePolicy.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.subway.path.domain;
+
+import lombok.Getter;
+
+@Getter
+public class DefaultFarePolicy implements FarePolicy{
+    private final Long DEFAULT_FARE = 1250L;
```

Comment:
상수를 의도하신 거라면, 관례에 따라 `static` 키워드도 추가해보면 어떨까요?
```suggestion
    private static final Long DEFAULT_FARE = 1250L;
```